docs/upload-your-snap.md

```diff
@@ -0,0 +1,55 @@
+# Upload to the store
+
+So you've been working hard on your .snap, and you finally have it to the point
+where you're ready to share it with the world? Great! You can use Snapcraft to
+upload it directly to the store.
+
+
+## Authenticate to the store
+
+First of all, you need to give Snapcraft permission to communicate with the
+store on your behalf:
+
+    $ snapcraft login
+    Enter your Ubuntu One SSO credentials.
+    Email: me@example.com
+    Password:
+    OTP: <press enter here if you haven't enabled two-factor authentication>
+    Authenticating against Ubuntu One SSO.
+    Starting new HTTPS connection (1): login.ubuntu.com
```

Comment:
I'm working on prettying that up now as part of LP: [#1538692](https://bugs.launchpad.net/snapcraft/+bug/1538692).